src/main/java/uk/org/tombolo/importer/utils/JSONReader.java

```diff
@@ -192,6 +191,7 @@ public void setPrimaryNode(String primaryNode) {
         JsonReader reader = Json.createReader(is);
         JsonValue value = reader.read();
         convertTreeToHashMap(value, EMPTY_KEY);
+        getFlatJsonTree().add(individualSectionOfTree);
```

Comment:
Why this change?

Reply:
because if you follow the logic, of how it gets and adds the value in final flat structure. The program return without adding the last section of the json file because of line 250, thus to add the final section of the json file, this step is necessary, it was bug in previous implementation which i realised while doing that issue